projects/rulenode-core-config/src/lib/components/transform/script-config.component.html

```diff
@@ -35,7 +35,7 @@
     </button>
   </tb-js-func>
   <div fxLayout="row">
-    <button mat-button mat-raised-button color="primary" (click)="testScript()">
+    <button mat-button mat-stroked-button color="primary" (click)="testScript()">
```

Comment:
Styles for the test script function button should be consistent between different nodes. The test script button for the filter script is raised.